pkgs/applications/science/logic/nusmv/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NuSMV";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://nusmv.fbk.eu/distrib/NuSMV-${version}-linux64.tar.gz";
+    sha256 = "1370x2vwjndv9ham5q399nn84hvhm1gj1k7pq576qmh4pi12xc8i";
+  };
+
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  installPhase = ''
+    mkdir $out
+    ls bin
```

Comment:
Why?

Reply:
Sorry I had forgotten to remove that from debugging the build